kube/deploy.sh

```diff
@@ -0,0 +1,42 @@
+export KUBE_NAMESPACE=${KUBE_NAMESPACE}
```

Comment:
is KUBE_NAMESPACE supposed to be getting passed in as well as the environment?
It'd mean doing something like
`drone build promote -p KUBE_NAMESPACE=cs-qa ukhomeoffice/hocs-toolbox 0 qa`
instead of
`drone build promote ukhomeoffice/hocs-toolbox 0 cs-qa`

Reply:
Done